helpers.go

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
Not a big fan of `helpers.go` as a name, it leads to creating a pile to throw out without thinking about proper naming. Especially if all functions are nicely related to k8s object references. Maybe `k8s_refs.go`, `references.go` etc

Reply:
Agreed.